src/main/java/org/gridsuite/modification/server/modifications/GenerationDispatch.java

```diff
@@ -268,8 +286,25 @@ public void onUpdate(Identifiable identifiable, String attribute, String variant
             if (identifiable.getType() == IdentifiableType.GENERATOR &&
                 attribute.equals("targetP") &&
                 Double.compare((double) oldValue, (double) newValue) != 0) {
-                report(reporter, suffixKey, "GeneratorSetTargetP", "Generator ${generator} targetP : ${oldValue} MW --> ${newValue} MW",
-                    Map.of(GENERATOR, identifiable.getId(), "oldValue", oldValue, "newValue", newValue), TypedValue.INFO_SEVERITY);
+                report(reporter, suffixKey, "GeneratorSetTargetP", "The active power set point of generator ${generator} has been set to ${newValue} MW",
+                    Map.of(GENERATOR, identifiable.getId(), "newValue", newValue), TypedValue.TRACE_SEVERITY);
+                updatedGenerators.add(identifiable.getId());
+            }
+        }
+
+        public void endReport(List<Generator> adjustableGenerators) {
+            report(reporter, suffixKey, "TotalGeneratorSetTargetP", "The active power set points of ${nbUpdatedGenerator} generator${isPlural} have been updated as a result of generation dispatch",
+                    Map.of("nbUpdatedGenerator", updatedGenerators.size(), "isPlural", updatedGenerators.size() > 1 ? "s" : ""), TypedValue.INFO_SEVERITY);
+            // what are unchanged generators ?
+            adjustableGenerators.stream()
+                .filter(g -> !updatedGenerators.contains(g.getId()))
+                .forEach(g -> report(reporter, suffixKey, "GeneratorUnchangedTargetP", "Generator ${generator} has not been selected by the merit order algorithm. Its active power set point has been set to 0",
+                        Map.of(GENERATOR, g.getId()), TypedValue.TRACE_SEVERITY));
+            int nbUnchangedGenerators = adjustableGenerators.size() - updatedGenerators.size();
+            if (nbUnchangedGenerators > 0) {
```

Comment:
Add this synthetic report before each individual unchanged generator report ?

Reply:
also made for update generators (synthetic before details)